atsign-foundation/at_widgets
# fix: lint issues and depreciated methods bugs fixed in relevant at_widgets's packages

**- What I did**

- Removed at_client_mobile_dependency override.
- updated at_client_mobile to 3.2.17 in at packages depending on it.
- AtAuthService and it's methods replaces the depreciated AtClientService and it's methods.
- KeyChainManager methods replaces the depreciated KeyChainUtil methods

**- How I did it**

**- How to verify it**

1. Run packages example app 
2. App should run bug free and without lint issues.
**- Description for the changelog**

Flutter analyzer issues resolved in all at_widgets's packages.
